src/c++/perf_analyzer/client_backend/triton/triton_client_backend.cc

```diff
@@ -590,6 +590,19 @@ TritonClientBackend::ParseInferOptionsToTriton(
   }
   triton_options->triton_enable_empty_final_response_ =
       options.triton_enable_empty_final_response_;
+
+  for (auto& map_entry : options.request_parameters_) {
+    auto rp = tc::RequestParameter();
+    rp.str_value = map_entry.second.str_value;
+    rp.int_value = map_entry.second.int_value;
+    rp.bool_value = map_entry.second.bool_value;
+    uint64_t val =
+        static_cast<std::underlying_type<cb::RequestParameterType>::type>(
```

Comment:
The 3 assignments above this:

![gif1](https://github.com/triton-inference-server/client/assets/7692737/a7b14c7a-8a3b-42ed-b974-303317360962)

This assignment:

![gif2](https://github.com/triton-inference-server/client/assets/7692737/5670ec6f-e3d9-4bcc-8ae4-c0a235039faf)

Reply:
I recognize the jank. I have a story to clean this up based on pushing the parsing of the options down to the grpc client. 
Hilarious but I need to clean this up.